src/main/java/dev/slimevr/VRServer.java

```diff
@@ -321,6 +325,28 @@ public void resetTrackersYaw() {
 		});
 	}
 
+	public void setLegTweaksEnabled(boolean value) {
+		queueTask(() -> {
+			humanPoseProcessor.setLegTweaksEnabled(value);
+		});
+	}
+
+	public void setSkatingReductionEnabled(boolean value) {
+		config.setProperty("legTweaks.skatingCorrection", value);
```

Comment:
Why not use a SkeletonConfig toggle? I don't get what manually managing the config for this adds.

Reply:
It didn't add anything, I changed it to using a Skeleton Config toggle